src/headless-cms/query.ts

```diff
@@ -72,12 +72,23 @@ export const PAGE_QUERY = gql`
     slug
     title
     uri
+    lead
     language {
       code
       slug
       locale
       name
     }
+    featuredImage {
+      node {
+        mediaItemUrl
+        link
+        altText
+        mimeType
+        title
+        uri
+      }
+    }
```

Comment:
Do we need this at all times?

Reply:
I thought it would just be simpler if it is. I also thought that if the page has a featured image, the page-view if anything, should show the picture. Then it would already be included in listing and the details page. Sure it won't be used in menus, but multiple different kind of field fragments are hard to maintain too.